client/changelist/change.go

```diff
@@ -38,8 +38,10 @@ type TufChange struct {
 // TufRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TufRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	ReplaceKeys data.KeyList `json:"keys"`
```

Comment:
Actually wait, I misread -- this is fine since this is the original `Keys` field :+1: